src/GPEmulator.jl

```diff
@@ -95,42 +116,81 @@ function GPObj(inputs, data, package::GPJL; GPkernel::Union{K, KPy, Nothing}=not
         sqrt_inv_input_cov = convert(Array{FT}, sqrt(inv(cov(inputs, dims=1))))
         inputs = (inputs .- input_mean) * sqrt_inv_input_cov
     end
-        
+
+    # Transform the outputs
+    sqrt_singular_values_inv = Diagonal(1.0 ./ sqrt.(SVD.S)) 
```

Comment:
Should `sqrt.(SVD.S)` be `sqrt.(decomposition.S)`?

Reply:
Done